### ISSUE TITLE
signature mismatch if `imagor-signer-truncate` not between 26 and 28

### ISSUE THREAD

**Issue by jonataw** [cshum/imagor#94]
Version: `0.9.10`

It appears that the path parser only detects hashes with a length between 26 and 30 characters as seen below:

https://github.com/cshum/imagor/blob/5dff3f60994b66a1d7d9e309fa152c896d2f02c4/imagorpath/parse.go#L15

Further, it checks if that segment is less or equal to 28 characters:

https://github.com/cshum/imagor/blob/5dff3f60994b66a1d7d9e309fa152c896d2f02c4/imagorpath/parse.go#L67-L71

Is this intended?

[The example here](https://github.com/cshum/imagor#custom-hmac-signer) sets `imagor-signer-truncate` to `32`, but that will not work with the behavior above.

**Comment by cshum** [comment]
Thanks for raising it, that is definitely a bug.

**Comment by cshum** [comment]
Fixed in `0.9.11`
```
docker run --rm -p 8000:8000 ghcr.io/cshum/imagor:0.9.11 -imagor-signer-type sha256 -imagor-signer-truncate 40 -imagor-secret 1234
```
Example URL:
```
http://localhost:8000/pOmOlw07XZQidA3hijbBvrXHKUCAye2CLlXS1yst/fit-in/-180x180/10x10/filters:hue(290):saturation(100):fill(yellow)/raw.githubusercontent.com/cshum/imagor/master/testdata/gopher.png
```
